eahub/profiles/management/commands/profilesfromlegacyhub.py

```diff
@@ -7,6 +7,8 @@
 from django.utils import html, timezone
```

Comment:
I wonder whether we can get rid of this code. The migration from legacy hub has happened, not sure it's worth our effort to maintain this code

Reply:
I agree here, I've been thinking about scheduling a 1-2 week-long cleanup once we're done with the pending tasks and have a bit of time. Ie to follow the best practice of refactoring sprints. 

I would say the only reason I haven't been advocating for one is that we're still in the project feasibility assessment stage.